src/components/TransactionItem.tsx

```diff
@@ -19,40 +21,109 @@ const renderCofirmations = (txType: TxType, confirmations: number) =>
     <Text style={styles.label}>{`${i18n.transactions.list.conf}: ${getConfirmationsText(txType, confirmations)}`}</Text>
   );
 
-export const TransactionItem = ({ item, onPress }: { item: Transaction; onPress: (item: any) => void }) => (
-  <TouchableOpacity style={styles.container} onPress={() => onPress(item)}>
-    <View style={styles.leftColumn}>
+export const TransactionItem = ({ item, onPress }: { item: Transaction; onPress: (item: any) => void }) => {
+  const isMinusValue = item.valueWithoutFee < 0;
```

Comment:
I'm not convinced that a component should have a logic of checking whether the number is negative or positive. As far as I can see it's being used only to determine whether a minus character should be displayed or not. 

I'd suggest developing a new object or class, let's name it `Btc`, which would have methods as follows:
- `addNegativeSign()`
- `addMissingZerosToSatoshis()`
- `addCurrencyCode()`

this way it'd be possible to display an amount however we like invoking, example: `new Btc(amount).addNegativeSign(). addMissingZerosToSatoshis(). addCurrencyCode().toString()`. 

It'd be a more testable approach as well.

Reply:
`isMinusValue` is used also for checking `isCanceledAlertToMe` and in `{ color: isMinusValue ? palette.textRed : palette.textBlack }`. Seems to be a little bit overhead with this approach, because we would need to add duplicate it many times for each amount. Adding a simple function that has all of this included inside is enough for me and I don't see any complication with writing unit test for `formatToBtcv`

Reply:
makes sense.

Reply:
with this approach, you would never use and test a function that uses other functions which I think is not the case. You would always combine all of the pure functions in place of usage which would make a lot of repetition in code